packages/manager/apps/web/client/app/domain/anycast/terminate/terminate.html

```diff
@@ -0,0 +1,22 @@
+<div>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span
+            data-translate="domain_configuration_dnsanycast_terminate_title"
+        ></span>
+    </oui-back-button>
+    <p
+        data-translate="domain_configuration_dnsanycast_terminate_description1"
+    ></p>
+    <p
+        data-translate="domain_configuration_dnsanycast_terminate_description2"
+    ></p>
+    <oui-button
+        class="mt-4"
+        data-on-click="$ctrl.terminate()"
```

Comment:
It looks like to be an anchor and not a button.

In that way we will be able to get rid of the `$state` injection in the controller and prefer using a `$state.href` through the resolve object.

Is that make sense to you?